### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Startpage](https://hosted.weblate.org/projects/jamulus/1-index/).

It also includes the following components:

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/General strings](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/Include-Server-Commands](https://hosted.weblate.org/projects/jamulus/include-server-commands/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/Navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)

Current translation status:

Weblate translation status

### CODE REVIEW

**Comment by hoffie** [comment]
@ignotus666, @ann0see, @pljones: Hrm, this PR contains three commits and I don't see authorship information in any of them. :(

Is there still a problem with the Github integration / squash configuration?

**Comment by ann0see** [comment]
It's configured that way. 

The reason behind that was that personal email gets leaked. As the the main repo has the checkbox enabled, probably enable it here too?

**Comment by trebmuh** [comment]
Remarks/questions:
- it's a bit strange that a bunch of (French translation) files get there "last translator" field updated while there is no update in the file
- in a bunch of other software I'm translating (or have been involved in), when a translator update a file, his name is defined as the "last translator" and the name of his predecessor is moved on a line below. I like that way since it shows the "team" and it doesn't feel that the last translator is the only one. Could we envisage to have the same system here?

File: _translator-files/po/fr/Contribution.po [hunk 1]
```diff
@@ -60,7 +60,7 @@ msgstr "Si vous pensez qu'il manque de la documentation ou des informations ou q
 #. type: Plain text
 #: ../wiki/en/Contribution.md:15
 msgid "We use [Weblate](https://hosted.weblate.org/projects/jamulus/#languages) to translate our documentation. Read about our [documentation and translation process](https://github.com/jamulussoftware/jamuluswebsite/blob/release/README.md), and [introduce yourself](https://github.com/jamulussoftware/jamulus/discussions) if you’d like to get involved."
-msgstr "Nous utilisons [Weblate](https://hosted.weblate.org/projects/jamulus/#languages) pour traduire notre documentation. Lire à propos notre [processus de documentation et de traduction](https://github.com/jamulussoftware/jamuluswebsite/blob/release/README.md), et [présentez-vous](https://github.com/jamulussoftware/jamulus/discussions) si vous souhaitez vous impliquer."
+msgstr "Nous utilisons [Weblate](https://hosted.weblate.org/projects/jamulus/#languages) pour traduire notre documentation. Lisez le à propos de notre [processus de documentation et de traduction](https://github.com/jamulussoftware/jamuluswebsite/blob/release/README.md), et [présentez-vous](https://github.com/jamulussoftware/jamulus/discussions) si vous souhaitez vous impliquer."
```
Comment (trebmuh) [inline _translator-files/po/fr/Contribution.po]: @jujudusud : ce changement me semble bizarre. Je trouve que ça ne se lit pas bien en francais : "lisez le à propos de... " ? Je pense que la version précédente était meilleure.

**Comment by ignotus666** [comment]
@trebmuh 

> it's a bit strange that a bunch of (French translation) files get there "last translator" field updated while there is no update in the file
in a bunch of other software I'm translating (or have been involved in), when a translator update a file, his name is defined as the "last translator" and the name of his predecessor is moved on a line below. I like that way since it shows the "team" and it doesn't feel that the last translator is the only one. Could we envisage to have the same system here?

I'm not sure why weblate decides to update just the header section and count that as an update by "x" translator. I can only presume it's because @jujudusud opened those files in weblate even if he didn't actually edit them.

As for your second point, I've been poking around the settings and I can't see anything related. I'll try to explore it further when I get a chance.

**Comment by ignotus666** [comment]
Another mystery is why weblate suddenly takes it upon itself to wrap text when it has a plugin enabled explicitly to prevent this. It's done it to the Korean version of "Running a Server" but none others...

**Comment by hoffie** [comment]
> in a bunch of other software I'm translating (or have been involved in), when a translator update a file, his name is defined as the "last translator" and the name of his predecessor is moved on a line below. I like that way since it shows the "team" and it doesn't feel that the last translator is the only one. Could we envisage to have the same system here?

Could that be this addon? https://docs.weblate.org/en/latest/admin/addons.html#contributors-in-comment

**Comment by ignotus666** [comment]
> Could that be this addon? https://docs.weblate.org/en/latest/admin/addons.html#contributors-in-comment

Looks like it, yes! @ann0see do you see any problem with adding this? I ask because it includes contributors' emails.

**Comment by trebmuh** [comment]
Looks great, thanks for the nice finding @hoffie

**Comment by ann0see** [comment]
@ignotus666 as long as my E-Mail is not on the release branch and others are ok with it, feel free to manage/install it on Weblate.

**Comment by ignotus666** [comment]
Done.

**Comment by ann0see** [comment]
> Another mystery is why weblate suddenly takes it upon itself to wrap text when it has a plugin enabled explicitly to prevent this. It's done it to the Korean version of "Running a Server" but none others...

Maybe not every component has it enabled?

**Comment by ignotus666** [comment]
I checked when I installed this last add-on and yes, it's enabled in all of them. Besides, it would have done it to all languages in that component. I just updated the Spanish translation of "Running a Server" and there's no wrapping. It's weird.

File: _translator-files/po/fr/Contribution.po (hunk 1 above)
Comment (jujudusud) [reply _translator-files/po/fr/Contribution.po]: @trebmuh , je propose :
"Découvrez notre processus...".

**Comment by henkdegroot** [comment]
This PR fixes #850

**Comment by ann0see** [comment]
Attention: I will merge this in about one hour.

**Comment by ann0see** [comment]
Merged.

**Pull request merged**